### PR TITLE
Add 'instantiation' property to security scheme

### PR DESCRIPTION
**Pull request by vcharpenay**
(duplicate with #1288, but the latter is out-dated)

### CODE REVIEW

**Comment by vcharpenay** [comment]
@Murloc6, @ethieblin is the change correct?

**Comment by vcharpenay** [comment]
commit 06a13d5 propagates changes of #1167 in this PR.

**Comment by ethieblin** [comment]
Except for the minor comment on the generated HTML, this looks great to me :)

**Review by sebastiankb — APPROVED** [review]

**Comment by vcharpenay** [comment]
> Except for the minor comment on the generated HTML, this looks great to me :)

could you elaborate? I don't see what you're referring to.

**Review by ethieblin — COMMENTED** [review]
sorry, i missed the "send review" button after writing my comment

**Comment by sebastiankb** [comment]
from today's TD call
- group is ok to merge
- [comment](https://github.com/w3c/wot-thing-description/pull/1289#discussion_r754948187) from @ethieblin should be addressed before
- if ready please merge the PR

File: ontology/td.html
Comment (vcharpenay) [reply ontology/td.html]: great, thanks. It should be fixed now. Can you double-check? I'll wait until you approve the PR.

**Review by ethieblin — APPROVED** [review]
Fine with me :)

**Pull request merged**
